data/changelog.yml

```diff
@@ -1635,3 +1635,19 @@
   Status: Released
   Entry: |
     - Bug fix: some addons not working after Scratch April Fools ended
+
+- Version: v1.38.0
+  Index: 92
+  Tag: v1.38.0
+  Name: v1.38.0
+  Date: 2024-05-14
+  Status: Released
+  Entry: |
+    - Various changes to support Manifest V3, the new extension architecture required by browsers
+    - "Editor dark mode and customizable colors" addon: now styles alerts (such as "project could not save")
+    - Bug fix: "enhanced full screen" causing stage to become blurry
+  Highlights:
+    Description: |
+      This is an experimental update that we're releasing to a portion of our user base. It contains some behind-the-scenes changes to support the new extension architecture required by browsers (Manifest V3). If you encounter any issues or unexpected behavior, please send feedback. There is also a button to send feedback in the settings page.

```

Comment:
I haven't looked, but is this text appears on the "Scratch Addons update notification"? If not, you can put it on the entry instead (reminder that the entry is a Markdown field).

Reply:
Ah, okay, I asked it since it sounds that it isn't. Disregard, then.